### DISCUSSION TITLE
Fix Explorer Crash Loop without Admin rights in Safe Mode

### DISCUSSION THREAD

**Discussion by citizen-gh** [valinet/ExplorerPatcher#1853] [Q&A] [answered]
Hi all, I have installed in 2022 ExplorerPatcher after a forced update to Windows 11 on my employers workstation.
Normally I have Admin rights but with BeyondTrust access elevation dialogue. No clue which version it was, I think sometime in August.
Last Friday after windows updates the Desktop/Explorer started crashing and restarting in endless loop. I though I might fix it in safe mode. However this got things even worse. In safe mode BeyondTrust doesn't display the admin rights elevation dialogue, hence no admin rights available. Now I can't even leave the safe mode, because without admin rights (BeyondTrust dialogue not being displayed) I can't edit msconfig. Also cmd:/bcedit doesn't work.

The laptop has bitlocker, but I have the recovery key. I have attached the SSD to another PC and renamed both ExplorerPatcher folders (in Program Files and in my User/Roaming), however explorer still crashes, which I assume is because the Registry has some ExplorerPatcher entries which need to be edited or removed. Can someone tell me how can I fix this issue? Should I edit the registry and which entries to remove to stop explorer from crashing? That's the only option I see before

Thank you in advance

**Comment by pyrates999** [comment]
see here for a possible answer: https://github.com/valinet/ExplorerPatcher/discussions/1345

But after that, you still need to properly uninstall explorer patcher using admin rights.

**Reply by citizen-gh** [reply, marked as answer]
Thank you! The specific step was to rename dxgi.dll which made the explorer function again and the desktop loaded normally.
However, I have now a different problem, because I can't boot back into normal mode due to BeyondTrust not woking in safe mode. But this is a BeyondTrust issue now. :)

**Reply by pyrates999** [reply]
Please mark it as the answer then.
